components/Post/PostContent.tsx

```diff
@@ -172,6 +172,11 @@ const PostContent = ({ post }: Props) => {
       </div>
       {/* <Reactions /> This future might be added later */}
```

Comment:
We planned to remove this component, right?

Reply:
Yes, forgot to remove the comment, sorry